plugins/base/src/main/kotlin/renderers/html/HtmlRenderer.kt

```diff
@@ -95,7 +95,6 @@ open class HtmlRenderer(
             }
             node.dci.kind in setOf(ContentKind.Symbol) -> div("symbol $additionalClasses") {
                 childrenCallback()
-                if (node.hasStyle(TextStyle.Monospace)) copyButton()
```

Comment:
This is the only real change in this PR. The rest is either formatting or to make tests pass